src/main/scala/mesosphere/marathon/storage/StorageConfig.scala

```diff
@@ -196,6 +198,7 @@ object CuratorZk {
       maxConcurrent = config.int("max-concurrent-requests", 32),
       maxOutstanding = config.int("max-concurrent-outstanding", Int.MaxValue),
       maxVersions = config.int("max-versions", StorageConfig.DefaultMaxVersions),
+      scanBatchSize = config.int("scan-batch-size", StorageConfig.DefaultScanBatchSize),
```

Comment:
Why is here a third `scan-batch-size` parameter which is neither `gc_scan_batch_size` nor `gc-actor-scan-batch-size`?